database_interaction.py

```diff
@@ -0,0 +1,12 @@
+import User
+import json
+def write(u):
+ database_file=open(".gitignore","r+")
```

Comment:
no man! maybe I didnt explain it well.
.gitignore is a git file that includes all the ignored files from tracking changes. (try to google it)
keep the database name as you want but the file database.txt shouldn't be uploaded with the PR

Reply:
haha i under stand now